package.json

```diff
@@ -1555,6 +1555,7 @@
                 "jupyter.changeDirOnImportExport": {
                     "type": "boolean",
                     "default": false,
+                    "deprecationMessage": "This setting is deprecated and will be removed in the next release.",
```

Comment:
I would have expected code changes as well? I assume we're still using this somewhere? Or are you thinking we'd do the actual removal next month?

Reply:
yes, removing next month was the plan. It still works for translating python files into notebooks on non-Windows, so leaving that functionality while marking deprecated seems like the best way to do it